kortex_driver/CMakeLists.txt

```diff
@@ -5,10 +5,19 @@ project(kortex_driver)
 add_compile_options(-std=c++11)
 add_definitions(-D_OS_UNIX)
 
-# custom bash script to download the API if it has not been downloaded yet
-execute_process(COMMAND ./download_kortex_api.bash
-                WORKING_DIRECTORY ../src/ros_kortex/kortex_api/scripts
-                RESULT_VARIABLE DOWNLOAD_API_RESULT)
+
+find_path (KORTEX_API_DOWNLOADER_PATH  # Set this variable
+			download_kortex_api.bash  # To the Working dir of this file 
+			${PROJECT_SOURCE_DIRl}/../kortex_api/scripts # Search this file here
+			../src/ros_kortex/kortex_api/scripts) # and here
```

Comment:
I tried it myself and it worked, but in fact I wonder if the absolute path line is useful. 
{PROJECT_SOURCE_DIR} will always be path/to/catkin_ws/src/whatever/additional/paths/kortex_driver, so I believe this absolute path could be deleted. It worked on my side (without this line), with my files being in :
- src/
- src/ros_kortex
- src/whatever/ros_kortex

Reply:
Hello !

Yeah you are right, I kept the code for no valid reason.
I made a new commit and I corrected a type on the "PROJECT_SOURCE_DIRl" variable (should be PROJECT_SOURCE_DIR)